h2o-algos/src/test/java/hex/rulefit/RuleFitUtilsTest.java

```diff
@@ -144,24 +148,19 @@ public void extractRulesFromTreeFromModelTesCategorical() {
             Set<Rule> treeRules =  Rule.extractRulesFromTree(sharedTreeSubgraph, 0);
             assertEquals(treeRules.size(), 8);
 
-            Condition condition1 = new Condition(0, Condition.Type.Categorical, Condition.Operator.In, -1.0, new String[] {"f1995", "f1996", "f1997", "f1998", "f1999", "f2000"}, new int[] {8, 9, 10, 11, 12, 13}, "fYear", false);
```

Comment:
@koniecsveta I changed the expectations - I found that the natural language representation is easier to work with, please let me know if you see an issue

Reply:
LGTM